divspl/main.py

```diff
@@ -22,7 +22,8 @@ def _bytes(x):
     if len(argv) > 1:
         with open(argv[1], 'r') as f:
             result = parser.parse(lexer.lex(f.read()))
-            os.write(1, _bytes(result.eval()))
+            for number in result.eval():
+                os.write(1, _bytes(number))
```

Comment:
I think I'd prefer `line` over `number` as the variable name here because what we return from `eval()` is `lines`.

We're also losing the newlines that were originally in `eval()`, so the output is appearing all on one line now. I think we can just append a `\n` here and it should be fine.